R/primary-keys.R

```diff
@@ -16,12 +16,13 @@
 #' @param force Boolean, if `FALSE` (default), an error will be thrown if there is already a primary key
 #'   set for this table.
 #'   If `TRUE`, a potential old `pk` is deleted before setting a new one.
+#' @param autoincrement If `TRUE`, a sequence of integers (from 1 to number of
+#'   rows) will be generated in column specified in `columns`.
```

Comment:
Can we also add an "experimental" badge for this argument?

```suggestion
#' @param autoincrement If `TRUE`, the  column specified in `columns` will be populated
#'   automatically with a sequence of integers.
```

Reply:
done in [0086bf3](https://github.com/cynkra/dm/pull/1654/commits/0086bf3e6f3155e8c4c4a504699d24397d9f5019)